### PR TITLE
Migrate to the new cache API for emoji mod

### PR DESCRIPTION
**Pull request by dylangarcia**
https://discordjs.guide/popular-topics/partials.html#enabling-partials

Beep boop I'm a robot.

### CODE REVIEW

**Comment by vcarl** [comment]
I'll see if I can test this tonight. What's the behavior that this should fix? Reacting on message that were sent before the bot was started?

**Comment by dylangarcia** [comment]
@vcarl this will make the emoji mod work period, there was an API change

File: features/emojiMod.js [hunk 1]
```diff
@@ -25,16 +25,16 @@ const reactionHandlers = {
     if (!isStaff(member)) {
       return;
     }
-    if (isStaff(message.guild.members.get(message.author.id))) {
+    if (isStaff(message.guild.members.cache.get(message.author.id))) {
       return;
     }
-    const usersWhoReacted = reaction.users.map(user =>
-      message.guild.members.get(user.id)
+    const usersWhoReacted = reaction.users.cache.map(user =>
```
Comment (nickmccurdy) [inline features/emojiMod.js]: cache is a Map, not an Array

File: features/emojiMod.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ const cooldown = require("./cooldown").default;
 const staffRoles = ["mvp", "moderator", "admin", "admins"];
 
 const isStaff = member =>
-  (member.roles || []).some(role =>
+  (member.roles.cache || []).some(role =>
```
Comment (nickmccurdy) [inline features/emojiMod.js]: cache is a Map, not an Array

File: features/emojiMod.js [hunk 3]
```diff
@@ -25,16 +25,16 @@ const reactionHandlers = {
...
-    const usersWhoReacted = reaction.users.map(user =>
-      message.guild.members.get(user.id)
+    const usersWhoReacted = reaction.users.cache.map(user =>
+      message.guild.members.cache.get(user.id)
     );
     const numberOfTotalReactions = usersWhoReacted.length;
     const numberOfStaffReactions = usersWhoReacted.filter(isStaff).length;
 
-    const modLogChannel = bot.channels.find(
+    const modLogChannel = bot.channels.cache.find(
```
Comment (nickmccurdy) [inline features/emojiMod.js]: cache is a Map, not an Array

File: features/emojiMod.js [hunk 4]
```diff
@@ -110,7 +110,7 @@ const reactionHandlers = {
 
     const numberOfTotalReactions = reactions.count;
 
-    const modLogChannel = bot.channels.find(
+    const modLogChannel = bot.channels.cache.find(
```
Comment (nickmccurdy) [inline features/emojiMod.js]: cache is a Map, not an Array

**Comment by dylangarcia** [comment]
The cache is the new way their API works. I'll merge and test it live 😛

**Pull request merged**